### PR TITLE
Improving note

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6341

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/sign-in-teams.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Teams apps on Windows and MacOS are gaining support for device policies that res
 When this policy is configured on a device, users can only sign in with accounts homed in an Azure AD tenant that is included in the “Tenant Allow List” defined in the policy. The policy applies to all sign-ins, including first and additional accounts. If your organization spans multiple Azure AD tenants, you can include multiple Tenant IDs in the Allow List. Links to add another account may continue to be visible in the Teams app, but they won't be operable.
 
 > [!NOTE]
->1. The policy only restricts sign-ins. It does not restrict the ability for users to be invited as guest in other Azure AD tenants, or switch to other tenants.
+>1. The policy only restricts sign-ins. It does not restrict the ability for users to be invited as guest in other Azure AD tenants, or switch to those other tenants (where users have been invited as guest).
```
Comment (JohanFreelancer9) [inline Teams/sign-in-teams.md:117]:
```suggestion
>1. The policy only restricts sign-ins. It does not restrict the ability for users to be invited as a guest in other Azure AD tenants, or switch to those other tenants (where users have been invited as a guest).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@MSFTTracyP PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
Can we merge this one ? :)

**Comment by get-itips** [comment]
@MSFTTracyP Can you review and merge? Thanks

**Comment by get-itips** [comment]
Replaced with https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/6969

**Pull request closed without merging**